### ISSUE TITLE
Capybara

### ISSUE THREAD

**Issue by pachadotdev** [ropensci/software-review#645]
Submitting Author Name: Mauricio Vargas Sepulveda
Submitting Author Github Handle: @pachadotdev
Repository:  https://github.com/pachadotdev/capybara
Version submitted:
Submission type: Standard
Editor:  TBD 
Reviewers:  TBD 

Archive: TBD
Version accepted: TBD
Language: en

---

-   Paste the full DESCRIPTION file inside a code block below:

```
Package: capybara
Type: Package
Title: Fast and Memory Efficient Fitting of Linear Models With High-Dimensional
    Fixed Effects
Version: 0.5.1
Authors@R: c(
    person(
        given = "Mauricio",
        family = "Vargas Sepulveda",
        role = c("aut", "cre"),
        email = "m.sepulveda@mail.utoronto.ca",
        comment = c(ORCID = "0000-0003-1017-7574"))
    )
Imports:
    dplyr,
    Formula,
    generics,
    magrittr,
    MASS,
    rlang,
    stats
Suggests: 
    fixest,
    knitr,
    rmarkdown,
    testthat (>= 3.0.0),
    tidyr
Depends: R(>= 3.5.0)
Description: Fast and user-friendly estimation of generalized linear models with
    multiple fixed effects and cluster the standard errors. The method to obtain
    the estimated fixed-effects coefficients is based on Stammann (2018) 
    <https://arxiv.org/abs/1707.01815> and Gaure (2013)
    <https://dx.doi.org/10.1016/j.csda.2013.03.024>.
License: Apache License (>= 2)
BugReports: https://github.com/pachadotdev/capybara/issues
URL: https://pacha.dev/capybara/, https://github.com/pachadotdev/capybara
LazyData: true
RoxygenNote: 7.3.1
Encoding: UTF-8
NeedsCompilation: yes
LinkingTo: cpp11, cpp11armadillo
VignetteBuilder: knitr
Config/testthat/edition: 3
Remotes: 
    pachadotdev/cpp11armadillo
    ropenscilabs/srr
Roxygen: list(markdown = TRUE, roclets = c("namespace", "rd", "srr::srr_stats_roclet"))
```

## Scope

- Please indicate which category or categories from our [package fit policies](https://ropensci.github.io/dev_guide/policies.html#package-categories) this package falls under: (Please check an appropriate box below. If you are unsure, we suggest you make a pre-submission inquiry.):

	- [ ] data retrieval
	- [ ] data extraction
	- [ ] data munging
	- [ ] data deposition
        - [ ] data validation and testing
	- [ ] workflow automation
	- [ ] version control
	- [ ] citation management and bibliometrics
	- [ ] scientific software wrappers
	- [ ] field and lab reproducibility tools
	- [ ] database software bindings
	- [ ] geospatial data
	- [ ] text analysis
	- [x] Other:  Regression and Supervised Learning

- Explain how and why the package falls under these categories (briefly, 1-2 sentences):

This helps to estimate linear models with many fixed effects and implement and efficient algorithm that saves time and memory. In addition, this features a novel wrapper for Armadillo (C++).

-   Who is the target audience and what are scientific applications of this package?

People (mostly) in the social sciences that need multiple controls in their models. This is especially useful in Economics and International Relations.

-   Are there other R packages that accomplish the same thing? If so, how does yours differ or meet [our criteria for best-in-category](https://ropensci.github.io/dev_guide/policies.html#overlap)?

-   (If applicable) Does your package comply with our [guidance around _Ethics, Data Privacy and Human Subjects Research_](https://devguide.ropensci.org/policies.html#ethics-data-privacy-and-human-subjects-research)?

Fixest, alpaca. This one has different design choices and a reduced number of dependencies. One attribute is that it adds multiple unit tests.

-   If you made a pre-submission inquiry, please paste the link to the corresponding issue, forum post, or other discussion, or @tag the editor you contacted.

https://github.com/ropensci/software-review/issues/629

-   Explain reasons for any [`pkgcheck` items](https://docs.ropensci.org/pkgcheck/) which your package is unable to pass.

## Technical checks

Confirm each of the following by checking the box.

- [x] I have read the [rOpenSci packaging guide](https://devguide.ropensci.org/building.html).
- [x] I have read the [author guide](https://devdevguide.netlify.app/authors-guide.html) and I expect to maintain this package for at least 2 years or to find a replacement.

This package:

- [x] does not violate the Terms of Service of any service it interacts with.
- [x] has a CRAN and OSI accepted license.
- [x] contains a [README with instructions for installing the development version](https://ropensci.github.io/dev_guide/building.html#readme).
- [x] includes [documentation with examples for all functions, created with roxygen2](https://ropensci.github.io/dev_guide/building.html#documentation).
- [x] contains a vignette with examples of its essential functions and uses.
- [x] has a [test suite](https://ropensci.github.io/dev_guide/building.html#testing).
- [x] has [continuous integration](https://ropensci.github.io/dev_guide/ci.html), including reporting of test coverage.

## Publication options

- [x] Do you intend for this package to go on CRAN?
- [ ] Do you intend for this package to go on Bioconductor?

- [ ] Do you wish to submit an Applications Article about your package to [Methods in Ecology and Evolution](http://besjournals.onlinelibrary.wiley.com/hub/journal/10.1111/(ISSN)2041-210X/)? If so:

<details>
<summary>MEE Options</summary>

- [ ] The package is novel and will be of interest to the broad readership of the journal.
- [ ] The manuscript describing the package is no longer than 3000 words.
- [ ] You intend to archive the code for the package in a long-term repository which meets the requirements of the journal (see [MEE's Policy on Publishing Code](http://besjournals.onlinelibrary.wiley.com/hub/journal/10.1111/(ISSN)2041-210X/journal-resources/policy-on-publishing-code.html))
- (*Scope: Do consider MEE's [Aims and Scope](http://besjournals.onlinelibrary.wiley.com/hub/journal/10.1111/(ISSN)2041-210X/aims-and-scope/read-full-aims-and-scope.html) for your manuscript. We make no guarantee that your manuscript will be within MEE scope.*)
- (*Although not required, we strongly recommend having a full manuscript prepared when you submit here.*)
- (*Please do not submit your package separately to Methods in Ecology and Evolution*)

</details>

## Code of conduct

- [x] I agree to abide by [rOpenSci's Code of Conduct](https://ropensci.org/code-of-conduct/) during the review process and in maintaining my package should it be accepted.

**Comment by pachadotdev** [comment]
@ropensci-review-bot check package

**Comment by jooolia** [comment]
Dear @pachadotdev , Thank you for your submission. 

You have checked "Regression and Supervised Learning" which would entail a "Statistical software review" (a different issue template to be opened, but that is ok). Would you be willing to go through the statistical software review as described here and implementing the standards:https://stats-devguide.ropensci.org/pkgdev.html

Thanks, Julia

**Comment by pachadotdev** [comment]
> Dear @pachadotdev , Thank you for your submission.
> 
> You have checked "Regression and Supervised Learning" which would entail a "Statistical software review" (a different issue template to be opened, but that is ok). Would you be willing to go through the statistical software review as described here and implementing the standards:https://stats-devguide.ropensci.org/pkgdev.html
> 
> Thanks, Julia

thanks a lot !

indeed, when I opened the issue that was the only available template on my end

I already implemented https://stats-devguide.ropensci.org/pkgdev.html#pkgdev-srr

**Comment by pachadotdev** [comment]
@ropensci-review-bot check package

**Comment by pachadotdev** [comment]
@jooolia Hi, I cannot find the statistical review template.

**Comment by jooolia** [comment]
Hi @pachadotdev , sorry for the delayed response. You have the option of several different types of templates when you open an issue the statistical one is the last choice (the template is viewable here: https://github.com/ropensci/software-review/blob/main/.github/ISSUE_TEMPLATE/F-submit-statistical-software-for-review.md). Does this help? Thanks, Julia

**Comment by pachadotdev** [comment]
thanks @jooolia 

here is the filled template

---
name: Capybara
about: Linear models with fixed effects

---

Submitting Author Name: Pacha
Submitting Author Github Handle: @pachadotdev 
Repository: https://github.com/pachadotdev/capybara
Version submitted: 0.2.5 (initially)
Submission type: Stats
Badge grade: silver (?)
Editor: @jooolia 
Reviewers: <TBD
Archive: TBD
Version accepted: TBD
Language: en
---



-   Paste the full DESCRIPTION file inside a code block below:

```
Package: capybara
Type: Package
Title: Fast and Memory Efficient Fitting of Linear Models With High-Dimensional
    Fixed Effects
Version: 0.5.2
Authors@R: c(
    person(
        given = "Mauricio",
        family = "Vargas Sepulveda",
        role = c("aut", "cre"),
        email = "m.sepulveda@mail.utoronto.ca",
        comment = c(ORCID = "0000-0003-1017-7574"))
    )
Imports:
    dplyr,
    Formula,
    generics,
    magrittr,
    MASS,
    rlang,
    stats
Suggests: 
    knitr,
    rmarkdown,
    testthat (>= 3.0.0),
    tidyr
Depends: R(>= 3.5.0)
Description: Fast and user-friendly estimation of generalized linear models with
    multiple fixed effects and cluster the standard errors. The method to obtain
    the estimated fixed-effects coefficients is based on Stammann (2018) 
    <https://arxiv.org/abs/1707.01815> and Gaure (2013)
    <https://dx.doi.org/10.1016/j.csda.2013.03.024>.
License: Apache License (>= 2)
BugReports: https://github.com/pachadotdev/capybara/issues
URL: https://pacha.dev/capybara/, https://github.com/pachadotdev/capybara
LazyData: true
RoxygenNote: 7.3.1
Encoding: UTF-8
NeedsCompilation: yes
LinkingTo: cpp11, cpp11armadillo
VignetteBuilder: knitr
Config/testthat/edition: 3
Remotes: 
    pachadotdev/cpp11armadillo,
    ropenscilabs/srr
Roxygen: list(markdown = TRUE, roclets = c("namespace", "rd", "srr::srr_stats_roclet"))
```

## Scope 

- Please indicate which of our [statistical package categories](https://stats-devguide.ropensci.org/overview.html#overview-categories) this package falls under. (Please check one or more appropriate boxes below):

     **Statistical Packages**

	- [ ] Bayesian and Monte Carlo Routines
	- [ ] Dimensionality Reduction, Clustering, and Unsupervised Learning
	- [ ] Machine Learning
	- [x] Regression and Supervised Learning
	- [ ] Exploratory Data Analysis (EDA) and Summary Statistics
	- [ ] Spatial Analyses
	- [ ] Time Series Analyses
	- [ ] Probability Distributions

## Pre-submission Inquiry

- [x] A pre-submission inquiry has been approved in [issue#629](#629)


## General Information


-   Who is the target audience and what are scientific applications of this package?

People (mostly) in the social sciences that need multiple controls in their models. This is especially useful in Economics and International Relations.

-   Paste your responses to our [*General Standard* **G1.1** here](https://stats-devguide.ropensci.org/standards.html#general-standards), describing whether your software is:

    - *An improvement on other implementations of similar algorithms in* **R**.

    Please include hyperlinked references to all other relevant software. https://github.com/amrei-stammann/alpaca/

## Badging

-    What grade of badge are you aiming for? ([silver](https://stats-devguide.ropensci.org/pkgdev.html#pkgdev-badges))

-    If aiming for silver or gold, describe which of the [four aspects listed in the *Guide for Authors* chapter](https://stats-devguide.ropensci.org/pkgdev.html#pkgdev-silver) the package fulfils (at least one aspect for silver; three for gold)

* Have a demonstrated generality of usage beyond one single envisioned use case.
* Compliance with a good number of standards beyond those identified as minimally necessary. 

## Technical checks

Confirm each of the following by checking the box.

- [x] I have read the [rOpenSci packaging guide](https://devguide.ropensci.org/building.html).
- x] I have read the [author guide](https://devdevguide.netlify.app/authors-guide.html) and I expect to maintain this package for at least 2 years or have another maintainer identified.
- [x] I/we have read the [*Statistical Software Peer Review* Guide for Authors](https://stats-devguide.ropensci.org/pkgdev.html).
- [x] I/we have run [`autotest`](https://github.com/ropensci-review-tools/autotest) checks on the package, and ensured no tests fail.
- [x] The [`srr_stats_pre_submit()` function](https://ropensci-review-tools.github.io/srr/reference/srr_stats_pre_submit.html) confirms this package may be submitted.
- [x] The [`pkgcheck()` function](https://docs.ropensci.org/pkgcheck/reference/pkgcheck.html) confirms this package may be submitted - alternatively, please explain reasons for any checks which your package is unable to pass.

This package:

- [x] does not violate the Terms of Service of any service it interacts with.
- [x] has a CRAN and OSI accepted license.
- [x] contains a [README with instructions for installing the development version](https://devguide.ropensci.org/building.html#readme).

## Publication options

- [x] Do you intend for this package to go on CRAN?
- [ ] Do you intend for this package to go on Bioconductor?


## Code of conduct

- [x] I agree to abide by [rOpenSci's Code of Conduct](https://ropensci.org/code-of-conduct/) during the review process and in maintaining my package should it be accepted.

**Comment by pachadotdev** [comment]
hi @jooolia 
I think the software is ready for review as is

**Comment by jooolia** [comment]
Dear @pachadotdev , thanks for your patience. We are discussing with the statistical editors and will get back to you shortly.
Thanks, Julia

**Comment by jooolia** [comment]
Dear @pachadotdev, thanks for updating to address the statistical aspects of the package. Currently there are some srrstatsTODO tags in your srr-stats-standards.R file and we note that "software to be submitted for review must contain no @srrstatsTODO tags" (https://stats-devguide.ropensci.org/pkgdev.html#pkgdev-srr). 

Could you also edit your first comment in this issue so that it contains the comment here https://github.com/ropensci/software-review/issues/645#issuecomment-2209395422  (e.g. replace the content with the statistical issue content? ) 

Thanks, Julia

**Comment by pachadotdev** [comment]
> Dear @pachadotdev, thanks for updating to address the statistical aspects of the package. Currently there are some srrstatsTODO tags in your srr-stats-standards.R file and we note that "software to be submitted for review must contain no @srrstatsTODO tags" (https://stats-devguide.ropensci.org/pkgdev.html#pkgdev-srr).
> 
> Could you also edit your first comment in this issue so that it contains the comment here [#645 (comment)](https://github.com/ropensci/software-review/issues/645#issuecomment-2209395422) (e.g. replace the content with the statistical issue content? )
> 
> Thanks, Julia

thanks!

I updated the comment. I will need 3-4 weeks to finish all the srrstatsTODO

**Comment by jooolia** [comment]
Ok thanks @pachadotdev for the update. Another editor will take over from me in few days to continue helping with the process. Cheers, Julia

**Comment by adamhsparks** [comment]
Hi, @pachadotdev, I'm taking over where @jooolia is leaving off as EIC. Please let me know what assistance you may need from me.

**Comment by pachadotdev** [comment]
hi @adamhsparks 
I think the pkg is ready for review

**Comment by adamhsparks** [comment]
@ropensci-review-bot check package

**Comment by maelle** [comment]
@ropensci-review-bot check package

**Comment by maelle** [comment]
@pachadotdev thanks for your submission! I'm fixing the content of your issue as you deleted HTML comments. Please do not delete them, they help the bot find the necessary metadata.

**Comment by maelle** [comment]
@ropensci-review-bot check package

**Comment by adamhsparks** [comment]
Thanks, @maelle.

@pachadotdev, could you please address the issues flagged with 'srr'?

**Comment by pachadotdev** [comment]
> Thanks, @maelle.
> 
> @pachadotdev, could you please address the issues flagged with 'srr'?

thanks @maelle and @adamhsparks 

I shall check the ssr issues

**Comment by pachadotdev** [comment]
@ropensci-review-bot check package

**Comment by pachadotdev** [comment]
@ropensci-review-bot check package

**Comment by pachadotdev** [comment]
hi @adamhsparks 
I fixed the check issues

The only thing I am not sure about is this: ✖️ All applicable standards [v0.2.0] have been documented in this package (99 complied with; 17 N/A standards)
Statistical standards should be documented in most package files, yet are mostly only documented in one file.

I tried different combinations and srr always creates a single file with the review tasks

**Comment by adamhsparks** [comment]
> hi @adamhsparks I fixed the check issues
> 
> The only thing I am not sure about is this: ✖️ All applicable standards [v0.2.0] have been documented in this package (99 complied with; 17 N/A standards) Statistical standards should be documented in most package files, yet are mostly only documented in one file.
> 
> I tried different combinations and srr always creates a single file with the review tasks

@mpadge, can you provide any insight here?

**Comment by mpadge** [comment]
@pachadotdev [Our *Stats Devguide* refers to this "single file" and states](https://stats-devguide.ropensci.org/pkgdev.html#pkgdev-srr):

> The first task after having generated this file is to move standards to approximate locations within your package where they are likely to be addressed. For example, standards concerning tests should be moved somewhere within the tests/ directory, standards concerning documentation to the main README.Rmd file, or within a vignette file...
>
> Moving different standards to more appropriate locations within your code will break down an initially large single list of standards into more manageable groups dispersed throughout your code. As each standard is addressed, it should be moved to one or more locations in your code as near as possible to relevant code, the tag changed from @srrstatsTODO to @srrstats, and a brief description appended to explain how that standard is addressed.

See the [main `srr` vignette](https://ropensci-review-tools.github.io/srr/articles/srr-stats.html) referred to in the *Stats Devguide* for more detail. For each individual standard, both general and category-specific, you need to describe how your code complies with the standard, *and move the standard itself* to the point or points within the code where that standard applies. Note that code will commonly adhere to single standards at multiple points, and that multiple copies of `@srrstats` tags are generally expected.

In short: The "srr-stats-standards.R" that is auto-generated by the `srr` package is just a template for you to use in *starting* to document how your code complies with each standard. From there, you have to work through each one, and modify the text to explain how you adhere to the standard.

**Comment by pachadotdev** [comment]
thanks @mpadge 
I re-organized the srr comments. The general aspects are still in the main srr file, while the specifics were moved to different scripts and therefore sometimes there is a duplication of the same tag (e.g., such as a `vcov` method in fileA.r and fileB.r)

**Comment by mpadge** [comment]
@ropensci-review-bot check srr

**Comment by mpadge** [comment]
@pachadotdev You can call `check srr` yourself at any time. The above output shows what you still need to do to document compliance. Please read the [main srr guide](https://docs.ropensci.org/srr/articles/srr-stats.html), and adhere to all requirements there. This includes documenting compliance in `/tests` files, as well as [standards for documentation](https://docs.ropensci.org/srr/articles/srr-stats.html#documenting-standards-for-documentation), both in `README.Rmd`, and in vignettes. Once you've done that, please use [`srr::srr_stats_pre_submit()`](https://docs.ropensci.org/srr/reference/srr_stats_pre_submit.html) locally to confirm compliance. Only once that indicates that everything is okay, feel free to call `check srr` here again to let us know. Thanks

**Comment by pachadotdev** [comment]
@ropensci-review-bot check srr

**Comment by pachadotdev** [comment]
sorry @mpadge , I must assume my dumminess now

I tried 

```
> srr::srr_stats_pre_submit() 
✔ All applicable standards [v0.2.0] have been documented in this package (190 complied with; 18 N/A standards)
! Standards should be documented in multiple directories, yet are only present in one
```

Even when I documented SRR standards here https://github.com/pachadotdev/capybara/blob/main/tests/testthat/test-feglm.R and in other tests, it still says that the tests are only in one directory

**Comment by pachadotdev** [comment]
> @ropensci-review-bot check srr

@ropensci-review-bot check srr

**Comment by mpadge** [comment]
Sorry @pachadotdev , the bot was down for a while. It's fixed again now, so I'll call the checks for you straight after this ...

**Comment by mpadge** [comment]
@ropensci-review-bot check srr

**Comment by mpadge** [comment]
Hmmm, and the :exclamation: is the one you mean, yet the text has been dropped. I'll check it out for you asap and report back. Thanks

**Comment by pachadotdev** [comment]
> Hmmm, and the ❗ is the one you mean, yet the text has been dropped. I'll check it out for you asap and report back. Thanks

it seems to the the srr pkg itself, let me test

**Comment by pachadotdev** [comment]
@mpadge the problem in srr is that it uses `base_path = "."` that combined with `gsub` goes into "Hulk" mode and removes all the paths

I proposed a fix here https://github.com/ropensci-review-tools/srr/pull/45/files#diff-c807a8e5a9d16e9a6c853c660ac8108f799a50d9a7c6f6c99ade0665f7caa28aR92

**Comment by mpadge** [comment]
@pachadotdev thanks, but that wasn't the issue; I think recent updates just hadn't yet been incorporated in the bot deployment. Everything seems fine, and i see you've updated the stats documentation, so feel free to call `check srr` again at any time.

**Comment by pachadotdev** [comment]
@ropensci-review-bot check srr

**Comment by pachadotdev** [comment]
@ropensci-review-bot check srr

**Comment by pachadotdev** [comment]
@mpadge Hi! I spreaded the documentation as you indicated. For most items I used the same wording from srr, because those were already clear. For some cases I explained with a new writing when it was pertinent for clarity.

**Comment by pachadotdev** [comment]
@ropensci-review-bot check srr

**Comment by pachadotdev** [comment]
@ropensci-review-bot check srr

**Comment by pachadotdev** [comment]
@ropensci-review-bot check srr

**Comment by pachadotdev** [comment]
@ropensci-review-bot check srr

**Comment by pachadotdev** [comment]
@ropensci-review-bot check srr

**Comment by pachadotdev** [comment]
@ropensci-review-bot check srr

**Comment by pachadotdev** [comment]
hi @mpadge 
I took a different approach now, I and proceeded to explain the code in the srr headers. Some explanations are too obvious but I did exactly what was requested. My approach was "not everybody has read Hansen's Econometrics, so it is not obvious how to pre-process the data"

**Comment by adamhsparks** [comment]
Hi, @pachadotdev, my time as EIC ended at the beginning of last month. Thank you for all your work to get {Capybara} to this point.

Pinging, @emilyriederer as the current acting EIC to action your submission.

**Comment by pachadotdev** [comment]
hi @emilyriederer :)

**Comment by emilyriederer** [comment]
Hi @pachadotdev ! Our general guidance is that authors should have one live review at a time to manage overall capacity. Do you want to move forward here and we'll put `{redatam}` on hold for now?

**Comment by pachadotdev** [comment]
> Hi @pachadotdev ! Our general guidance is that authors should have one live review at a time to manage overall capacity. Do you want to move forward here and we'll put `{redatam}` on hold for now?

yes, Lital has not responded yet, so redatam should wait

**Comment by emilyriederer** [comment]
@ropensci-review-bot check srr

**Comment by emilyriederer** [comment]
@ropensci-review-bot check package

**Comment by emilyriederer** [comment]
@ropensci-review-bot assign @rkillick  as editor

**Comment by emilyriederer** [comment]
@ropensci-review-bot help

**Comment by emilyriederer** [comment]
@ropensci-review-bot assign @rkillick as editor

**Comment by maurolepore** [comment]
Dear @pachadotdev and @rkillick just to let you know that my EiC rotation just started and I'm checking all open issues. 

Looks like the package passes all checks cleanly 💯 

@rkillick are you still able to handle this submission?

**Comment by rkillick** [comment]
@maurolepore Yes, I've been trying to find reviewers.

**Comment by maurolepore** [comment]
Great, thanks so much! I'll step back.

**Comment by pachadotdev** [comment]
> [@maurolepore](https://github.com/maurolepore) Yes, I've been trying to find reviewers.

hi @rkillick, can I send the package to CRAN in the meanwhile?

**Comment by maurolepore** [comment]
@pachadotdev while I'm here I'll share the relevant guideline:

> We strongly suggest submitting your package for review before publishing on CRAN or submitting a software paper describing the package to a journal. Review feedback may result in major improvements and updates to your package, including renaming and breaking changes to functions.
-- https://devdevguide.netlify.app/softwarereview_author


So I interpret this as we don't encourage it but also we won't stop you.

**Comment by pachadotdev** [comment]
> [@pachadotdev](https://github.com/pachadotdev) while I'm here I'll share the relevant guideline:
> 
> > We strongly suggest submitting your package for review before publishing on CRAN or submitting a software paper describing the package to a journal. Review feedback may result in major improvements and updates to your package, including renaming and breaking changes to functions.
> > -- https://devdevguide.netlify.app/softwarereview_author
> 
> So I interpret this as we don't encourage it but also we won't stop you.

thx, I will wait then :)

**Comment by rkillick** [comment]
@pachadotdev Apologies, but I'm really struggling to find reviewers for this package. Our guide allows authors to make suggestions of three potential reviewers.  Do you have any suggestions?

**Comment by maurolepore** [comment]
@pachadotdev I'm wrapping up my EiC rotation but before I hand on this submission to the next EiC I wanted to thank you for all your contributions to rOpenSci.

**Comment by emilyriederer** [comment]
Hi @rkillick and @pachadotdev -- I have just begun my EiC rotation and am checking in on stalled reviews. Are we still actively seeking reviewers for this submission? 

As a note mostly to myself, this open review was blocking beginning work on #672 . So, we shall revisit that issue once we clarify the path forward here.
